mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -134,7 +134,7 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 		ARG 2 inventory
 	METHOD method_7292 getLuck ()F
 	METHOD method_7293 vanishCursedItems ()V
-	METHOD method_7294 canModifyWorld ()Z
+	METHOD method_7294 canModifyBlocks ()Z
```

Comment:
```suggestion
	METHOD method_7294 canBreakBlocks ()Z
```
unless this is used for interaction checks like axe right click/shovel right click

Reply:
It's used for interaction checks where the interaction may modify the block, like inverting a daylight sensor.